### DISCUSSION TITLE
Exclude very small buildings from building type quest?

### DISCUSSION THREAD

**Discussion by smichel17** [streetcomplete/StreetComplete#2696] [Q&A] [answered]
Particularly if they are not close to a road. For example: https://www.openstreetmap.org/way/153661396

At least in my rural-ish area, these buildings are

- Usually sheds (not enough for me to call it spam, but close)
- Sometimes hard to answer the question, if they are not near the street (often behind the house in this case)
- Sometimes these are not actually buildings (boxes or trailers) or are buildings that SC does not include (tree houses).
- Sometimes they are no longer there (presumably boxes or trailers that have been moved)

**Comment by andrewharvey** [comment, marked as answer]
> Usually sheds (not enough for me to call it spam, but close)

They may well usually be sheds, but unless there is a way this could be assumed default, it's still a good idea to tag them so data consumers know this.

> Sometimes these are not actually buildings (boxes or trailers) 

Then it's a good idea that SC asks, so you can answer with a note "not a building, is a box" and then it can be updated to not be a building.

> Sometimes they are no longer there (presumably boxes or trailers that have been moved)

Again good reason for SC to ask, so you can answer no longer there and it can be updated.

**Reply by matkoniecz** [reply]
very related: https://github.com/streetcomplete/StreetComplete/issues/1439

> Again good reason for SC to ask, so you can answer no longer there and it can be updated.

+1

**Reply by smichel17** [reply]
Makes sense!

**Reply by joeldebruijn** [reply]
How do I fill in this situation.

Its a row of sheds belonging to a row of houses. Each house has one shed.

It isnt a seperate house, but "shed" isnt a type or category to choose from.

![Screenshot_20220428-071126.png](https://user-images.githubusercontent.com/3906940/165681521-ca3c2128-08a1-483a-a8fb-9a7beb113ed8.png)

**Reply by joeldebruijn** [reply]
Sorry found it 

![Screenshots_2022-04-28-07-23-56.png](https://user-images.githubusercontent.com/3906940/165682109-6d004003-1a2c-4d49-a718-0fc175d34899.png)
